app/src/main/java/com/example/fisrtapplication/fragments/DataListFragment.java

```diff
@@ -18,76 +21,73 @@
 import com.google.firebase.auth.FirebaseAuth;
 
 import java.util.List;
+import java.util.Objects;
 
 import androidx.annotation.NonNull;
-import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
 import androidx.recyclerview.widget.LinearLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
 import androidx.swiperefreshlayout.widget.SwipeRefreshLayout;
 import retrofit2.Call;
 import retrofit2.Callback;
 import retrofit2.Response;
 
-public class DataListActivity extends AppCompatActivity {
 
+public class DataListFragment extends Fragment {
     private RecyclerView recyclerView;
     private VendingsAdapter vendingsAdapter;
     private SwipeRefreshLayout refreshLayout;
     private FirebaseAuth mAuth;
     private ProgressBar progressBar;
     private View content;
+    private View rootView;
```

Comment:
Please get rid of it. You can either pass view to your `setupViews()` method or use `onViewCreated()` + `getView()` instead